spec/built-in-modules/math.md

```diff
@@ -486,4 +486,25 @@ This function is also available as a global function named `percentage()`.
 random($limit: null)
 ```
 
-This function is also available as a global function named `random()`.
+The `random()` function can take an optional parameter `$limit` which defaults
```

Comment:
You don't need to write this out explicitly; the signature above already specifies it.